.expeditor/scripts/shared.ps1

```diff
@@ -78,9 +78,20 @@ function Install-Rustup($Toolchain) {
   } else {
       Write-Host "Installing rustup and $toolchain-x86_64-pc-windows-msvc Rust."
       [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
-      invoke-restmethod -usebasicparsing 'https://static.rust-lang.org/rustup/dist/i686-pc-windows-gnu/rustup-init.exe' -outfile 'rustup-init.exe'
-      ./rustup-init.exe -y --default-toolchain $toolchain-x86_64-pc-windows-msvc --no-modify-path --profile=minimal
-      $env:path += ";$env:USERPROFILE\.cargo\bin"
+      try { 
+        Invoke-RestMethod -UseBasicParsing 'https://static.rust-lang.org/rustup/dist/i686-pc-windows-gnu/rustup-init.exe' `
+                          -OutFile 'rustup-init.exe' `
+                          -MaximumRetryCount 5 `
+                          -RetryIntervalSec 5 
+      } catch {
+        Write-Host "Unable to install rustup"
+        # Dig into the exception to get the Response details.
+        Write-Host "StatusCode:" $_.Exception.Response.StatusCode.value__ 
+        Write-Host "StatusDescription:" $_.Exception.Response.StatusDescription
+      }
+
+#./rustup-init.exe -y --default-toolchain $toolchain-x86_64-pc-windows-msvc --no-modify-path --profile=minimal
+#$env:path += ";$env:USERPROFILE\.cargo\bin"
```

Comment:
Should this be commented out?

Reply:
:man_facepalming: No it should not. I was testing in pwsh on Linux.  Fixed in 1614121bf842f5ea7cb531dfd8dceb2b4c9c8c31